Dockerfiles/ccp-openshift-scan/scanning/yumupdates.py

```diff
@@ -1,44 +1,117 @@
-# scan script for yum list update scannr
+# scan script for yum list update scanner
 
-import scan_lib
+import sys
+import yum
 
 
-def yum_updates():
+class SysStdoutSuppressor(object):
     """
-    Finds yum updates
+    Class which can be used to mute output printed
+    on sys.stdout.
+    It just mutes the sys.stdout and takes care of raising
+    the exception if encountered in calling method.
     """
-    command = ["yum", "-q", "check-update"]
-    out, err = scan_lib.run_cmd_out_err(command)
-    return out, err
 
+    def __enter__(self):
+        self.stdout = sys.stdout
+        sys.stdout = self
 
-def find_updates():
+    def __exit__(self, type, value, traceback):
+        # revert back sys.stdout to original form before exiting
+        sys.stdout = self.stdout
+        if type is not None:
+            raise(value)
+
+    def write(self, x):
+        """
+        Overriding sys.stdout.write, this method won't print
+        on stdout
+        """
+        pass
+
+
+class YumUpdates(object):
     """
-    process yum updates
+    Class with related methods to find the yum updates
+    available using python yum API client
```

Comment:
ok -
I am updating it to
```
    This class contains methods to find yum updates and print
    information of RPM updates.
    It uses yum python API client to find the updates.
```

Removed containers as this can also work on normal systems.